pgcli/pgexecute.py

```diff
@@ -50,7 +50,13 @@ def _wait_select(conn):
             conn.cancel()
             # the loop will be broken by a server error
             continue
-
+        except select.error as e:
+            errno, _ = e.args
```

Comment:
I'd use `errno = e.args[0]`, what's the advantage of _ ?

Reply:
You're right. I've fixed it.